core/src/main/scala/Manifest.scala

```diff
@@ -387,7 +387,9 @@ object Manifest {
    * place i could find to put it.
    */
   private def parseDeployable(release: Github.Release, name: String): Task[Deployable] = {
-    release.findAssetContent(s"${name}.deployable.yml").flatMap { a =>
+    release.findAssetContent(s"${name}.deployable.yml").handleWith {
+      case ProblematicDeployable(_, _) => release.findAssetContent(s"${name}.deployable.yaml")
+    }.flatMap { a =>
```

Comment:
Could just do the lazy:

```
case ProblematicDeployable(_, _) => release.findAssetContent(s"${name}.deployable.yaml or ${name}.deployable.yml")
```

Reply:
Aha that's an interesting hack, would cause a third useless traversal though the list though. Perhaps it's better to just leave it and decide if we care about `.yml` or `.yaml` more? :P

Reply:
@adelbertc how would it cause an extra traversal? I'm just suggesting altering the message to make it clear for users :-)

Reply:
The string argument is used [both to find the actual content and for the error message](https://github.com/getnelson/nelson/blob/master/core/src/main/scala/Github.scala#L93), so you can't do what you suggested without either breaking the actual find-the-asset bit
or doing a 3rd dummy traversal for the benefit of getting a better error message

Reply:
Ahhhh sorry i'm being slow. Yeah not ideal I guess - would love something to indicate to the users as this is one of the most common messages first-time users will see. 

Perhaps as an alternitive we could simply augment the underlying error message? https://github.com/getnelson/nelson/blob/master/core/src/main/scala/Exceptions.scala#L47 - how about:

```
NelsonError(s"Error when attempting to parse deployable from repository. Please ensure that a valid deployable yaml/yml file is available at the specified URL, or attached to the associated Github release: ${url} -- $str")
```